src/main/java/fi/aalto/cs/apluscourses/intellij/actions/SubmitExerciseAction.java

```diff
@@ -334,18 +334,31 @@ private void trySubmitTutorial(@NotNull Project project,
     TutorialExercise exercise = tutorialViewModel.getExercise();
 
     // a submission for an IDE activity only has one file with a magic file name
-    var tutorialResultFile = FileUtilRt.createTempFile("tutorials", null);
+    var tutorialResultFile = FileUtilRt.createTempFile(Tutorial.TUTORIAL_SUBMIT_FILE_NAME, null);
     String payload = tutorial.getSubmissionPayload();
     FileUtils.writeStringToFile(tutorialResultFile, payload, StandardCharsets.UTF_8);
 
-    Map<String, Path> files = Map.of(Tutorial.TUTORIAL_SUBMIT_FILE_NAME, tutorialResultFile.toPath());
+    var submissionInfo = exercise.getSubmissionInfo();
+    String language = "en";
+
+    if (!submissionInfo.isSubmittable(language)) {
+      logger.warn("{} not submittable", exercise);
```

Comment:
Make the string `"Tutorial {} not submittable"` because the same string is used in submitting normal exercises - otherwise we won't be able to distinguish the two